### DISCUSSION TITLE
Dataset used for training your models

### DISCUSSION THREAD

**Discussion by AlbertoPresta** [tensorflow/compression#172] [Q&A]
Hi, 

maybe it is written in the documentation...did you use CLIC dataset to train your models? 

Alberto

**Comment by jonaballe** [comment]
Hi Alberto,

for the models accessible through tfci.py, the dataset used to train them is described in the accompanying paper. Mostly, they are trained on a large dataset of web-scraped images (not CLIC).

Hope this helps!

**Reply by AlbertoPresta** [reply]
Thanks! is this dataset pubicly available? does this dataset have a name?

**Reply by jonaballe** [reply]
It is not, unfortunately.

**Reply by AlbertoPresta** [reply]
thanks for answering. What do you mean with "large dataset"? above 100k?
